### PR TITLE
HW_6 done

### PR DESCRIPTION
**Pull request by dubinchuk**
Please review HW_6

### CODE REVIEW

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

File: HW_6/task 6.3.2.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    for (let i = 0; i < wordArr.length; i++) {
+        if (wordArr[i] == wordArr[i - 1]) {    
+            i -= 2;
+        }
+        if ((wordArr[i] == wordArr[i + 1]) && wordArr[i] == alphabet[alphabet.length - 1]) {
+            wordArr.splice(i, 2, alphabet[0]);
+            i--;
+        }
+        if (wordArr[i] == wordArr[i + 1]) {
+            replacementLetterIndex = alphabet.indexOf(wordArr[i]) + 1;
```
Comment (VladimirLQA) [inline HW_6/task 6.3.2.js:25]:
```suggestion
           const replacementLetterIndex = alphabet.indexOf(wordArr[i]) + 1;
```

File: HW_6/task 6.3.1.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// Task 3
+// 5*. Напишите рекурсивную функцию, которая принимает на вход число и складывает его цифры. 
+//   Если сумма получилась больше 9 - снова сложите цифры. И так пока, сумма не станет меньше либо равной 9. 
+//   После окончания сложений возвращает полученное число. Например при подаче числа 19 (1+9=10>9, потому 1+0=1) выводится 1
+
+function sum(number) {
+    if (!(/\d/.test(number) && typeof number == 'number')) {
+        throw new Error('Введите корректные данные');
+    }
```
Comment (VladimirLQA) [inline HW_6/task 6.3.1.js:9]:
```suggestion
    if (typeof number !== 'number')) {
        throw new Error('Введите корректные данные');
    }
```

File: HW_6/task 6.2.3.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    throw new Error('Введите корректные данные');
+}
+
+console.log(uniqueNumbers([3, 5, 5, 7, 324, 54, 5, 3, 52525, 4536, 5, 78, 324, 4536]))
+
+
+// решение с использованием Spread оператора:
+
+function uniqueNumbers2(numbers) {
+    if (typeof numbers == 'object') {
```
Comment (VladimirLQA) [inline HW_6/task 6.2.3.js:31]:
```suggestion
    if (Arrary.isArray(numbers)) {
```

File: HW_6/task 6.2.1.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        if (!competitorPizzas.includes(pizza)){
+            result.push(pizza);
+        }
+    }
+    
+    if (result == '') {
+        return null;
+    } 
+    
+    return result;
```
Comment (VladimirLQA) [inline HW_6/task 6.2.1.js:26]: You've declared `result` as an array and after checking if it's empty string, hmmm
```suggestion
    return result.length ? result : null;
```

File: HW_6/task 6.1.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    const employees = ['Ivan Ivanov', 'Petr Petrov', 'Alexey Alexeev', 'Elena Elenova', 'Fedor Fedorov'];
+    const salary = ['2000', '1500', '1700', '2100', '2400'];
+    
+    for (let i = 0; i < employees.length; i++) {
+        if (employeeName == employees[i]) {
+            const employeeInfo = [];
+            employeeInfo.push(employees[i], salary[i]);
+            return employeeInfo;
```
Comment (VladimirLQA) [inline HW_6/task 6.1.js:23]:
```suggestion
            return [employees[i], salary[i]];
```

**Review by VladimirLQA — APPROVED** [review]

File: HW_6/task 6.2.3.js (hunk 3 above)
Comment (dubinchuk) [reply HW_6/task 6.2.3.js]: fixed

File: HW_6/task 6.1.js (hunk 5 above)
Comment (dubinchuk) [reply HW_6/task 6.1.js:23]: The suggested code is more compact, however the task was to create array and use `.push` method to add items into array

File: HW_6/task 6.3.1.js (hunk 2 above)
Comment (VladimirLQA) [reply HW_6/task 6.3.1.js:9]: `Напишите рекурсивную функцию, которая принимает на вход число и складывает его цифры.`

**Pull request merged**